### ISSUE TITLE
I have some issues in the early stages of reproducing the code.

### ISSUE THREAD

**Issue by The-fleeting-happy-moments** [JunyingWang959/Adv-CPG#2]
Hello, thank you for the open-source code. I have the following questions to ask you.

Question 1:
May I ask how your dataset was processed? Is it using the Consistent ID project author method you mentioned, that is, using ConsistentID/data/FGID.sh file?

Question 2:
May I ask how your dataset was selected?

You mentioned in the original text: 'Among them, FFHQ and CelebA-HQ are used for face verification and LFW are used for face identification. For face verification, 1000 images with different identities are randomly selected as source images and randomly divided into 5 groups for FFHQ and CelebA-HQ datasets. Meanwhile, we select 5 images for each group. As a result, each dataset form consists of 5 groups with 200 source target image pairs in each group. For face identification, 500 identities are randomly selected from the LFW and one image of each group.' Identity is put into the gallery set and another image is put into the probe set. The attack setting is similar to GIFT [27]

Is it convenient to provide the dataset samples you have selected? If possible, I would greatly appreciate it.

Question 3:
May I ask how long your training time is approximately?

You mentioned in the original text: 'Adam [24] is set as the optimizer, the learning rate is set to 10-4, and the training is performed on 4 NVIDIA 3090 GPUs with a batch size of 8. In addition, a 50 step DDIM [46] sampler is employed and the scale of the classifier guidance setting is set to 5.'

Due to insufficient GPU resources, I only have one 3090 available for use. I would like to know your running time to evaluate the cost issue. I would greatly appreciate it if you could provide an answer.
